pkg/integrations/postgres_exporter/postgres_exporter_test.go

```diff
@@ -0,0 +1,24 @@
+package postgres_exporter
```

Comment:
You (unfortunately) need to add `//nolint:golint` to the end of this line so the linter doesn't get upset, see the other file for reference